spec/mocks/responses/LegalEntityManagement/get_document.json

```diff
@@ -12,6 +12,7 @@
       }
   ],
   "description": "passportimagess",
+  "fileName": "passportimagess",
```

Comment:
Er zijn minor wijzigingen wanneer ik een 'echt' request uitvoerde. (zie ook in de files hieronder)